### DISCUSSION TITLE
Biotin as variable modification

### DISCUSSION THREAD

**Discussion by pejota66** [vdemichev/DiaNN#804] [General]
Hello,

I'd like to use biotin as a variable modification to analyze DIA data 
Please let me know what code I should use to add this info to my run.

kind regards

**Comment by pejota66** [comment]
hi @vdemichev, could you please follow-up on the point above?

many thanks.

**Reply by vdemichev** [reply]
Please use --var-mod with the respective mass.

**Reply by pejota66** [reply]
For biotin, shall i write something like:
--var-mod 226,K

many many thanks for all your help and continue the nice work with DIA-NN.

**Reply by vdemichev** [reply]
You need to specify the mass, please see the correct syntax in DIA-NN docs

**Reply by vdemichev** [reply]
One point: will need to add --strip-unknown-mods when generating a predicted spectral library

**Comment by mniu0405** [comment]
@pejota66 Hi, I have the same modification for my sample. Do you find the way to add this biotin modification? I tried your comand "--var-mod 226,K", but it is not working.

**Reply by vdemichev** [reply]
That is not the correct syntax, e.g. also need to specify the mass, please see the docs
